src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -87,6 +87,10 @@
          (icons/icon {:style {:color (:exception-state style/colors) :marginRight 5 :verticalAlign "middle"}}
                      :warning-triangle)
          (str "Warning: This will launch " wf-count " workflows")]]))
+   [:div {:style {:textAlign "right" :fontSize "80%"}} (style/create-link {:text  (str "Cromwell Version: " (:cromwell-version props))
```

Comment:
See my PR about suggested indentation. I also suggest moving this to the top of the modal.